wallet/wallet.go

```diff
@@ -823,9 +826,14 @@ func (w *Wallet) FetchHeaders(chainClient *chain.RPCClient) (count int,
 		if err != nil {
 			return
 		}
+	} else {
```

Comment:
i'm reneging on this, didn't realize it would require a second db transaction, which is not what we want.